apps/src/templates/tables/QuickActionsCell.jsx

```diff
@@ -30,9 +30,7 @@ const styles = {
 
 class QuickActionsCell extends Component {
   static propTypes = {
-    children: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.array]).isRequired,
+    children: PropTypes.node.isRequired,
```

Comment:
@caleybrock I did this to resolve this Circle error: https://circleci.com/gh/code-dot-org/code-dot-org/59174?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link It works as far as I can tell for the current uses of `QuickActionsCell`; do you think it could be problematic?

Reply:
I think it would need to be node or array, incase it uses multiple items. (Chatted offline)